app/views/standard_layout.scala.html

```diff
@@ -120,7 +120,7 @@
                     <li class="dropdown">
                         @partials.with_username { username =>
                             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-                                @username <span class="caret"></span>
+                                @if(username.startsWith("guest-")) { Guest } else { @username } <span class="caret"></span>
```

Comment:
would this be better done in the partial?

Reply:
Thanks changed.